newIDE/app/src/InstructionOrExpression/EnumerateExpressions.js

```diff
@@ -10,15 +10,6 @@ const gd: libGDevelop = global.gd;
 
 const GROUP_DELIMITER = '/';
 
-// Support enumerate expressions that can be string, number or both ("number|string").
```

Comment:
Are we sure we don't need it any more?
I think that it may be better to not suggest functions returning a string when the node is a number.

Reply:
Indeed, my bad 😅

Reply:
But, it's open to discussion, I guess.

Reply:
It makes sense not to open suggestions for numbers as all numbers can be converted to string while you have to make sure your string is a number before converting it.
Let's keep the filtering for number expressions for now and, if needs arise, we could remove this filtering later?

Reply:
I agree with you on this point, number -> string is the thing users are most likely to do so let's suggest those options, whereas string -> number is quite rare.

My coding background still urges me to see all the functions, regardless of the type, and then I decide what I use...
But our users may not have the same view.

Let's keep the filtering and we'll get more feedback about this :)

Reply:
Agree!